packages/transform/README.md

```diff
@@ -0,0 +1,57 @@
+<div align="center">
+
+![Josh Logo](https://evie.codes/josh-light.png)
+
+# @joshdb/transform
+
+**A data manipulation middleware for Josh.**
```

Comment:
Perhaps a more descriptive message here.

Reply:
Ah yes, I didn't know what to put so I did something similar to the others at the time. This will be the last thing I will do, as the others take priority 🙂